server/data/probationTeamsClient.js

```diff
@@ -21,6 +21,9 @@ module.exports = signInService => {
     async getFunctionalMailbox(probationAreaCode, lduCode, teamCode) {
       const ldu = await get(`${apiUrl}/probation-areas/${probationAreaCode}/local-delivery-units/${lduCode}`)
       const teamAddress = R.path(['probationTeams', teamCode, 'functionalMailbox'], ldu)
+      if (!(teamAddress || (ldu && ldu.functionalMailbox))) {
+        logger.info(`Could not find functional mailbox for team code: ${teamCode} or ldu code: ${lduCode}`)
```

Comment:
It would be helpful to add the probationAreaCode to the values that are logged.

Reply:
done